Makefile

```diff
@@ -14,8 +14,8 @@ COMPILE_TARGET=./tmp/_output/bin/$(PROJECT)
 PROMETHEUS_OPERATOR_VERSION=v0.34.0
 LOCAL=local
 GRAFANA_OPERATOR_VERSION=v3.2.0
-AMO_VERSION?=v1.1.6
-PREV_AMO_VERSION=v1.1.5
+AMO_VERSION=1.2.0
+PREV_AMO_VERSION=1.1.6
```

Comment:
@HubertStefanski the versions here used to begin with a 'v'. Is it okay that they no longer do? I'm just thinking that there might be make targets expecting the format to be the same or something

Reply:
I'm not quite sure, this was causing a few issues when I worked with it, with some resources being called `whatever.vv1.1.5` so that's why I removed it, previous releases, apart from the one that's diffed here hadn't prefixed the `v` either and seemingly worked fine :shrug:

Reply:
The releases and tag names still need the `v`.